src/main/scala/com/iheart/workpipeline/akka/patterns/queue/Queue.scala

```diff
@@ -141,30 +139,30 @@ trait Queue extends Actor with ActorLogging with MessageScheduler {
   def onQueuedWorkExhausted(): Unit = ()
 }
 
-
-case class QueueWithBackPressure(settings: BackPressureSettings,
-                                 defaultWorkSettings: WorkSettings = WorkSettings(),
-                                 metricsCollector: MetricsCollector = NoOpMetricsCollector) extends Queue {
+case class QueueWithBackPressure(
+  settings: BackPressureSettings,
+    defaultWorkSettings: WorkSettings = WorkSettings(),
+    metricsCollector: MetricsCollector = NoOpMetricsCollector
```

Comment:
might have to reformat this file also.

actually, maybe just do a `find . -exec touch {} \;` to recursively touch all files in the repo so scalariform knows they changed